openpaygo/__init__.py

```diff
@@ -1,9 +1,5 @@
-from .token_encode import OpenPAYGOTokenEncoder
```

Comment:
Why removing those imports? They are left at top level so that you can access them importing just the module, if we remove them it will break existing tools using it.

Reply:
It raised "unused variable" warnings. I think the recommended approach is to use `__all__`. Let me try to adapt. Do you have a piece of code, where I can test?

Reply:
Nothing public but basically you should be able to import all of those variables when importing openpaygo. Like `from openpaygo import TokenType` should work.

Reply:
I personally find it a bit odd, if `from openpaygo import TokenType` works, but `from openpaygo import *` and then `openpaygo.TokenType` doesn't. Using `__all__` should be non-breaking. WDYT?